.github/workflows/android.yml

```diff
@@ -0,0 +1,59 @@
+name: Android
+on:
+  push:
+    tags:
+      - v*
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  build:
+    name: Build for Android
+    runs-on: ubuntu-latest
+    steps:
+      - name: Mount bazel cache
+        uses: actions/cache@v1
+        with:
+          path: "/home/runner/.cache/bazel"
+          key: bazel
```

Comment:
we don't use bazel anymore

Reply:
ah ok, this was on the install script of buildkite :) (I havn't finished yet)